### ISSUE TITLE
Laravel Error: GET http://cabinet.laravel.com/latest.zip

### ISSUE THREAD

**Issue by BaseMax** [laravel/installer#257]
C:\Users\Max\LaravelAPI>laravel -V
```
Laravel Installer 1.3.3
```

C:\Users\Max\LaravelAPI>laravel new
```
Crafting application...

In RequestException.php line 113:

  Server error: `GET http://cabinet.laravel.com/latest.zip` resulted in a `520 ` response


new [--dev] [--5.2] [--] [<name>]
```

C:\Users\Max\LaravelAPI>laravel new --dev
```
Crafting application...

In RequestException.php line 113:

  Server error: `GET http://cabinet.laravel.com/latest-develop.zip` resulted in a `520 ` response


new [--dev] [--5.2] [--] [<name>]
```

http://cabinet.laravel.com/latest.zip

![image](https://user-images.githubusercontent.com/2658040/219932892-ce48d307-3e75-4051-885b-5c38efdb22aa.png)

**Comment by driesvints** [comment]
You’re using a much outdated version of the installer. Please upgrade to the latest version

**Comment by BaseMax** [comment]
Thanks Dries, Now I am tring to run `composer global require laravel/installer`:

![image](https://user-images.githubusercontent.com/2658040/219936975-3a1f4f5c-7389-4a05-97f6-095703dfae49.png)

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires laravel/installer ^4.4 -> satisfiable by laravel/installer[v4.4.0].
    - laravel/installer v4.4.0 requires symfony/console ^4.0|^5.0|^6.0 -> found symfony/console[v4.0.0, ..., v4.4.49, v5.0.0, ..., v5.4.19, v6.0.0, ..., v6.2.5] but the package is fixed to v3.4.47 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require laravel/installer:*" to figure out if any version is installable, or "composer require laravel/installer:^2.1" if you know which you need.

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

doc: https://laravel.com/docs/master
